### PR TITLE
First draft1

### PR DESCRIPTION
**Pull request by fishmanjohn**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const MapAndGems = styled.div`
+  display: flex;
+  height: 100vh;
+  min-height: 100vh;
+  `
+
+  const [[latitude, longitude], setLatLong] = useState([33.812468, -117.918989])
+  const [refresh, setRefresh] = useState(false);
+  const updatePosition = (latitude, longitude) => {
+    console.log("this is the lat and lng >>>>>>>>>>>", latitude, longitude);
```
Comment (mzstevens69) [inline src/App.js]: remove console.log  line 20

Comment (fishmanjohn) [reply src/App.js]: Got it!

File: src/components/createGem.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    const submitGem = () => {
+        props.setRefresh(!props.refresh);
+        props.updatePosition(Number(form.latitude), Number(form.longitude));
+    }
+
+    return (
+        <FormContainer>
+            <form onSubmit={(e) => {
+                e.preventDefault();
+                axios.post('https://geoseek-be-stage.herokuapp.com/api/gems', form)
```
Comment (mzstevens69) [inline src/components/createGem.js]: add env variable here

File: src/components/map.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+
+    useEffect(() => {
+        setViewport({...viewport, latitude, longitude})
+    }, [latitude, longitude])
+
+
+
+    useEffect(() => {
+        axios.get('https://geoseek-be-stage.herokuapp.com/api/gems')
```
Comment (mzstevens69) [inline src/components/map.js]: add env var here line 31

File: src/components/navbar.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    return (
+        <NavBar>
+            <a href='/'><img src={Logo} alt="Main Logo" /></a>
+            {/* <Router> */}
+            <div>
+                <button><Link className='link' to='/CreateGem'>Create a Gem</Link></button>
+                <button><Link className='link' to='/ViewGem'>View Gems</Link></button>
+            </div>
+
+            {/* <Route path = '/CreateGem' component = {CreateGem}/>
```
Comment (mzstevens69) [inline src/components/navbar.js]: remove this commented out code.

File: src/components/viewGem.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
+import React, {useState, useEffect} from 'react'
+import {Link} from "react-router-dom"
+import axios from 'axios'
+import GemCard from './gem'
+import styled from 'styled-components'
+
+export default function ViewGem ({updatePosition}) {
+    const [gems, setGems] = useState([])
+    useEffect(() => {
+        axios.get('https://geoseek-be-stage.herokuapp.com/api/gems')
```
Comment (mzstevens69) [inline src/components/viewGem.js]: env var for line 10

**Pull request merged**